.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "docs/neptune-client"]
 	path = docs/neptune-client
-	url = https://github.com/neptune-ml/neptune-client
+	url = https://github.com/neptune-ai/neptune-client
```

Comment:
@hubertjaworskineptune changed link to submodule. Ok?

Reply:
it should work IMO